### ISSUE TITLE
Black on black

### ISSUE THREAD

**Issue by seefood** [robertosartori/BlackTwitter#1]
I tried 1.1.0 and the feed looks right, but when I post or enter the settings, it's black on black.
are you planning on color pickers for full customization? otherwise these presets should be a usable default :)

**Comment by roberto-sartori-gl** [comment]
Yes, it is an old problem. I will look next month (i've no time now) but i don't think i can solve that, not with Xposed. For me this is not a problem, i never write on Twitter, but i understand that this is bad. Unfortunately if i change color to the feed background, the same color is used when you have to post something (or in the settings), but i will look better when i find time.
